src/main/java/org/fenixedu/academic/domain/accounting/Event.java

```diff
@@ -673,39 +673,58 @@ public void forceCancel(final Person responsible, final String cancelJustificati
         if (isCancelled()) {
             return;
         }
+        
         changeState(EventState.CANCELLED, new DateTime());
         super.setResponsibleForCancel(responsible);
         super.setCancelJustification(cancelJustification);
     }
 
-    public void cancel(final Person responsible, final String cancelJustification) {
+    public void exempt(Person responsible, String justification) {
+        DateTime when = new DateTime();
 
-        if (isCancelled()) {
-            return;
-        }
-        
-        if (!isOpen()) {
-            throw new DomainException("error.accounting.Event.only.open.events.can.be.cancelled");
+        DebtInterestCalculator debtInterestCalculator = getDebtInterestCalculator(when);
+        Money dueInterestAmount = new Money(debtInterestCalculator.getDueInterestAmount());
+        Money dueFineAmount = new Money(debtInterestCalculator.getDueFineAmount());
+        Money dueAmount = new Money(debtInterestCalculator.getDueAmount());
+
+        when = when.plusSeconds(1);
```

Comment:
I understand why we need this type of construct (to exempt an event we need exempt the interests/fines first) but this usage probably isn't needed here. 

The `when.plusSeconds(1)`are already properly managed in the ifs.